website/package.json

```diff
@@ -7,13 +7,13 @@
     "@hashicorp/mktg-global-styles": "2.1.0",
     "@hashicorp/nextjs-scripts": "16.2.0",
     "@hashicorp/react-button": "4.0.0",
-    "@hashicorp/react-docs-page": "10.9.4-alpha.18",
+    "@hashicorp/react-docs-page": "11.0.1",
     "@hashicorp/react-hashi-stack-menu": "^1.1.0",
     "@hashicorp/react-head": "1.1.6",
     "@hashicorp/react-inline-svg": "5.0.0",
     "@hashicorp/react-markdown-page": "^0.1.0",
     "@hashicorp/react-product-downloader": "4.0.2",
-    "@hashicorp/react-search": "^3.0.0",
+    "@hashicorp/react-search": "4.1.1-alpha.2",
```

Comment:
Indexing content relies on a function within search-tools that I'm proposing we split out, via https://github.com/hashicorp/react-components/pull/161. Will bump this to a stable pre-release once that PR is finalized, merged, and published.